modules/visual_script/editor/visual_script_property_selector.cpp

```diff
@@ -1122,9 +1122,8 @@ TreeItem *VisualScriptPropertySelector::SearchRunner::_create_class_item(TreeIte
 	String what = "Class";
 	String details = p_doc->name;
 	if (p_doc->category.begins_with("VisualScriptCustomNode/")) {
-		Vector<String> path = p_doc->name.split("/");
 		icon = ui_service->get_theme_icon(SNAME("VisualScript"), SNAME("EditorIcons"));
-		text_0 = path[path.size() - 1];
+		text_0 = p_doc->name.get_slice("/", p_doc->name.get_slice_count("/") - 1);
```

Comment:
I'm not sure that's equivalent because `get_slice_count("/")` can return 0 if `p_doc->name` is empty (btw this could reuse `details` defined above).

So this could end up with `-1`, while `split("/")` would always have 1 item.

So... this whole PR needs to be re-reviewed in light of this difference.

Reply:
Well, then `text_0` will be an empty String (`get_slice()` returns "" for -1), so it's not a problem.